lib/ansible/modules/network/junos/junos_command.py

```diff
@@ -137,8 +137,8 @@
 from ansible.module_utils.netcli import Conditional
 from ansible.module_utils.network_common import ComplexList
 
-def check_args(module, warnings):
-    junos_check_args(module, warnings)
+def check_args(module):
```

Comment:
to make change backwards compatible with modules not in repo, make `warnings=None`